int32 id
geometry_msgs/PoseStamped pose
shape_msgs/SolidPrimitive shape
---
bool success